independent-publisher-connectors/AmazonS3Bucket/apiProperties.json

```diff
@@ -0,0 +1,43 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "accessKeyId",
+          "description": "The accessKeyId for this api",
+          "tooltip": "Provide the accessKeyId",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "accessKeySecret",
+          "description": "The accessKeySecret for this api",
+          "tooltip": "Provide the accessKeySecret",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
IconBrandColor should be "#da3b01" and the apiProperties.json file format should be as mentioned here. - https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json

Reply:
I updated the iconBrandColor